src/utils/translation.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright the eleventy-plugin-fluid copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/eleventy-plugin-fluid/raw/main/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/eleventy-plugin-fluid/raw/main/LICENSE.md.
+*/
+import rosetta from "rosetta";
+
+/**
+ *
+ * @param {String} key - The translation key. Must exist in (at least) one language within the translations object.
+ * @param {Object} values - An object containing values which can be substituted for keyed placeholders in the translated string.
+ * @param {Object} data - An object containing the translations and locale keys from Eleventy's global data.
+ * @return {String} - The localized string.
+ */
+const __ = function (key, values = {}, data = {}) {
+    if (!data.hasOwnProperty("locale") || !data.hasOwnProperty("translations")) {
+        data = this.ctx;
+    }
```

Comment:
If a data  object like `{locale: "fr"}` is passed in, it will be ignored because currently this requires that both `locale` and `translations` be provided in the data object. Instead maybe accept either and provide defaults for each, like:

```javascript
const locale = data.locale || this.page.lang;
const translations = data.translations || this.ctx.translations;
```

Also note above that ElventyI18nPlugin provides the page language via `this.page.lang`.